x/storage/types/expected_keepers.go

```diff
@@ -74,10 +74,14 @@ type PermissionKeeper interface {
 
 type CrossChainKeeper interface {
 	GetDestBscChainID() sdk.ChainID
+	GetDestOpChainID() sdk.ChainID
+
 	CreateRawIBCPackageWithFee(ctx sdk.Context, chainID sdk.ChainID, channelID sdk.ChannelID, packageType sdk.CrossChainPackageType,
 		packageLoad []byte, relayerFee *big.Int, ackRelayerFee *big.Int,
 	) (uint64, error)
 
+	IsDestChainSupported(chainID sdk.ChainID) bool
+
 	RegisterChannel(name string, id sdk.ChannelID, app sdk.CrossChainApplication) error
```

Comment:
do we need to specify the dest chain id ?

Reply:
no, the app is bind to the channel, all the dest chains use the same app